spynnaker8/spinnaker.py

```diff
@@ -324,7 +324,7 @@ def is_a_pynn_random(self, thing):
         return isinstance(thing, RandomDistribution)
 
     def get_pynn_NumpyRNG(self):
-        return NumpyRNG()
+        return NumpyRNG
```

Comment:
this seems unrelated

Reply:
This is essential to the changes in sPyNNaker.  This allows us to re-run the random with the original seed, which is needed if connectivity is to be regenerated.

Reply:
k.